.github/dependabot.yml

```diff
@@ -6,11 +6,13 @@ updates:
     package-ecosystem: "npm"
     schedule:
       interval: "monthly"
+    open-pull-requests-limit: 0
```

Comment:
why this instead of just deleting the whole `package-ecosystem` block for the languages we don't care about?